src/DynamoCoreWpf/Interfaces/IShellCom.cs

```diff
@@ -198,7 +198,7 @@ interface IFileDialog : IModalWindow
         // Defined on IModalWindow - repeated here due to requirements of COM interop layer
         // --------------------------------------------------------------------------------
         [MethodImpl ( MethodImplOptions.InternalCall, MethodCodeType = MethodCodeType.Runtime ), PreserveSig]
-        int Show ( [In] IntPtr parent );
+        new int Show ( [In] IntPtr parent );
```

Comment:
these were copy pasted from http://www.pinvoke.net but I guess this change is ok